onnx/checker.py

```diff
@@ -8,8 +8,9 @@
 """
 
```

Comment:
```suggestion
from __future__ import annotations
```

I would add this to resolve https://dev.azure.com/onnx-pipelines/onnx/_build/results?buildId=45271&view=logs&j=26ad7b42-9ea7-5ad4-891d-e581c481314e&t=36dc6483-53ad-5f31-0c8e-c9104543cd82&l=1450